api/src/api/session.js

```diff
@@ -30,6 +30,46 @@ router.get(
   }),
 );
 
+//get all displayed sessions on the home page
+router.get(
+  '/displayed',
+  errorWrap(async (req, res) => {
+    let activeSessions = await Session.find({ active: 'true' });
+    if (activeSessions.length >= 10) {
```

Comment:
Create a variable for the number of displayed sessions. I used 10 in this issue as a placeholder, but this might change depending on Ellie's designs, so it would be easier to update the variable instead of all the places where you used "10" or "10 - something" in this endpoint.

Reply:
Cool! I called it 'max' and used a let initialization

Reply:
'max' is still a bit vague. Can you call it maxDisplayed or something similar that's more descriptive?

Reply:
Ahh I see. I've just pushed these changes!